### ISSUE TITLE
Pulmonary methacholine challenge test abnormal

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11173]
**Preferred term label:**
Pulmonary methacholine challenge test abnormal

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1096472
https://www.aaaai.org/tools-for-the-public/conditions-library/asthma/methacholine-challenge-test
"A methacholine challenge test is considered positive if methacholine causes the lung function (FEV1) to drop by 20% or more compared to your baseline."

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Asthma
COPD

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001277
- label: Abnormal methacholine challenge test
- parents: [Abnormality on pulmonary function testing (HP:0030878)](https://hpo.jax.org/app/browse/term/HP:0030878)
- pmid: PMID:31613496
- def: An abnormal result of the methacholine challenge test, which is a bronchoprovocation technique used to evaluate airway hyperresponsiveness, particularly in patients with suspected asthma.
- comment: Methacholine, a derivative of acetylcholine, directly induces bronchoconstriction by stimulating airway smooth muscle receptors. In some cases, hyperresponsiveness may be the only objective sign of asthma, making this test a valuable diagnostic tool. The procedure involves stepwise administration of methacholine, followed by spirometry to measure the forced expiratory volume in 1 second. This process continues until either a 20% reduction in the forced expiratory volume in 1 second or a 35-40% reduction in specific airways conductance is observed. The dose required to cause a 20% reduction is known as the provocation dose (PD20). A PD20 of 200 µg, or a provocation concentration (PC20) of 8 mg/mL or less, indicates a positive result (100 microgram or 4 mg/mL for specific airway conductance).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11173
- synonyms:
